cost/turbonomics/buy_reserved_instances_recommendations/aws/turbonomics_buy_reserved_instances.pt

```diff
@@ -0,0 +1,307 @@
+name "Turbonomics Buy Reserved Instances Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "Turbonomics policy for recommending purchasable reserved instances [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/buy_reserved_instances_recommendations/aws) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.0",
```

Comment:
Since this is a brand new policy I think version number should be 1.0, this version number needs to match with the one at CHANGELOG.md

Reply:
@Rangelmv23 Can you please confirm ?

Reply:
In my case for the azure long running instances I used 2.0 version in changelog and in the actual policy. I remember seeing that in AWS example.